File: PRAK108-2310817210001-ANDRABRAPUTRAAKBARSALEH.c
#include <stdio.h>

int main() {
    float a = 14;
    float b = 5;
    float c = 3.14 * 2;
    float d = a / b;
    float r = d / c;

    printf("Diketahui :\n");
    printf("Pak Dengklek mengelilingi taman = %.f Putaran\n", b);
    printf("Jarak tempuh Pak Dengklek = %.f Kilometer\n", a);
    
    printf("Jawaban :\n");
    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer\n", r);
    
    return 0;
}

File: PRAK108-2310817210001-ANDRABRAPUTRAAKBARSALEH.py
a = int("5")
b = int("14")
c = float("3.14")
d = b / a
r = d / (2 * c) 
hasil = "{:.2f}".format(r)

print("Diketahui : ")
print("Pak Dengklek mengelilingi taman =", a, "Putaran")
print("Jarak tempuh Pak Dengklek =", b, "Kilometer")

print("jawaban :")
print("Jari-jari taman yang dikelilingi Pak Dengklek adalah", hasil, "Kilometer")